test/Driver/linker-args-order-linux.swift

```diff
@@ -4,6 +4,5 @@
 print("hello world!")
 // RUN: %empty-directory(%t)
 // RUN: %target-swiftc_driver -driver-print-jobs -static-stdlib -o %t/static-stdlib %s -Xlinker --no-allow-multiple-definition 2>&1| %FileCheck %s
-// CHECK: {{.*}}/swift -frontend -c -primary-file {{.*}}/linker-args-order-linux.swift -target x86_64-unknown-linux-gnu -disable-objc-interop
```

Comment:
Instead of dropping this entire line, please replace the target being checked with a regex match, like the paths!

Reply:
That doesn't make sense, the whole point is to remove the target check. Why is it needed for this test? If this is an x86_64 specific test then it should be disabled on non x86_64 machines.

Reply:
This line isn't just checking the target; it's checking that the frontend command runs before the autolink-extract command, and that it's compiling the way we expect. I guess there's no real significance to `-disable-objc-interop` either, but the rest of it's definitely useful.

Reply:
Got it, frontend command check is back in.